iroha_client/tests/tests/tests.rs

```diff
@@ -0,0 +1,15 @@
+mod account_with_multiple_signatories;
```

Comment:
Wow `.../tests/tests/tests.rs` does this need to be this much nested?

Reply:
Unfortunately. May be we can rename it but I didn't come up with good name

Reply:
We need it as `tests/*.rs` files are treated by rust as separate binaries. What is why we need to have only 1 file in `tests` directory and many files nested.

Reply:
But shouldn't it be then `iroha_client/tests/tests.rs`
And other files in `iroha_client/tests/tests/..`?

Reply:
Ok, I see.